src/main/scala/com/fulcrumgenomics/bam/EstimatePoolingFractions.scala

```diff
@@ -0,0 +1,242 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016 Fulcrum Genomics LLC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.fulcrumgenomics.bam
+
+import com.fulcrumgenomics.FgBioDef._
+import com.fulcrumgenomics.cmdline.{ClpGroups, FgBioTool}
+import com.fulcrumgenomics.util.{Io, Metric, Sequences}
+import com.fulcrumgenomics.vcf.ByIntervalListVariantContextIterator
+import dagr.commons.util.LazyLogging
+import dagr.sopt.{arg, clp}
+import htsjdk.samtools.SamReaderFactory
+import htsjdk.samtools.util.SamLocusIterator.LocusInfo
+import htsjdk.samtools.util._
+import htsjdk.variant.variantcontext.VariantContext
+import htsjdk.variant.vcf.VCFFileReader
+import org.apache.commons.math3.stat.regression.OLSMultipleLinearRegression
+
+import scala.collection.JavaConversions.iterableAsScalaIterable
+import Math.{min, max}
+
+@clp(group=ClpGroups.SamOrBam, description=
+  """
+    |Examines a pooled sample and estimates the fraction of each constituent sample.
+    |Uses a VCF of known genotypes for the samples within the mixture along with a
+    |BAM of sequencing data derived from the pool.  Performs a multiple regression
+    |for the alt allele fractions at each SNP locus, using as inputs the individual
+    |sample's genotypes.  Only SNPs that are bi-allelic within the pooled samples are
+    |used.
+    |
+    |Various filtering parameters can be used to control which loci are used:
+    | --intervals will restrict analysis to variants within the described intervals
+    | --min-genotype-quality will filter out any site with any genotype with GQ < n
+    | --min-coverage-per-sample requires that the coverage of a site in the BAM be
+    |     >= min-coverage-per-sample * n_samples
```

Comment:
I don't disagree but I'm struggling to come up with a better name.  I want it to be (coverage / samples) so that a sensible default can be set, and you don't have to think about changing it for the number of samples in the pool.   `min-mean-coverage-per-sample` is that best I've got, and I still don't like it.